acceptance/tests/fixtures/bases/multiport-app/serviceaccount.yaml

```diff
@@ -7,3 +7,19 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: multiport-admin
+---
+apiVersion: v1
+kind: Secret
+metadata:
```

Comment:
Another question. Is there no way that we could generate these secrets for the user, like say in the webhook, when it is multi-port?

Reply:
I think we could, it wouldnt be a substantial amount of work, but I think it *kinda* goes in the face of the spirit of the change that Kubernetes was making here. Secondly, and more importantly, if we did that we'd need to update the RBAC for our `clusterrole` for the controller to have `write` on `secrets` resources.
Thoughts?

Reply:
I think it is probably not worth it right now. If we end up with more multi-port users in the future then maybe do it then. It sounds like a low priority backlog item to me.